inc/saihostif.h

```diff
@@ -397,20 +397,20 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_PIPELINE_DISCARD_ROUTER = 0x00007002,
 
-    /** Exception traps custom range start */
-    SAI_HOSTIF_TRAP_TYPE_CUSTOM_EXCEPTION_RANGE_BASE = 0x00008000,
```

Comment:
why remove this ? and not use it like this:
SAI_HOSTIF_TRAP_TYPE_MPLS_TTL_ERROR  = SAI_HOSTIF_TRAP_TYPE_CUSTOM_EXCEPTION_RANGE_BASE??

Reply:
because it is not a custom trap but an MPLS error. MPLS traps are given range from 0x8000 to 0x8fff. Do you see any issue with the existing change?

Reply:
It was not removed, only the value changed.